packages/contracts/src/current/protocol/TokenRegistry/TokenRegistry.sol

```diff
@@ -16,9 +16,9 @@
 
 */
 
-pragma solidity ^0.4.11;
+pragma solidity ^0.4.19;
```

Comment:
Let's keep the version and the imports the same until we actually make changes to the TokenRegistry.